packages/dashboard/src/components/template/elements/index.ts

```diff
@@ -0,0 +1,8 @@
+import './element-template.scss';
+
+export { default as AppToTradeCard } from './_app-to-trade-card';
+export { default as Banner } from './_banner';
+export { default as Pair } from './_pair';
+export { default as Indice } from './_indice';
+export { default as AppToTradeBenefit } from './_app-to-trade-benefit';
+export { default as YouMightLikeCard } from './_you-might-like-card';
```

Comment:
Please remove this, let's only export from the component specific folder, not from elements.